docs/pages/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -198,7 +212,7 @@ is emitted to the Audit Log.
 
 ## What else does the Teleport FIPS binary enforce?
 
-- TLS protocol version is restricted to TLS 1.2.
+- TLS protocol version is restricted to TLS 1.2 and TLS 1.3.
 - All uses of non-compliant algorithms such as NaCl are removed and replaced with compliant algorithms such as AES-GCM.
 - Teleport is compiled with [BoringCrypto](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4407).
```

Comment:
Does boring 20220613 have a new CMVP certificate yet?

Reply:
It does not. Upstream Go decided to go ahead and swap to it in advance because it's close enough in the CMVP review process and since they wanted to comply with [NIST SP 800-52 Rev. 2](https://csrc.nist.gov/pubs/sp/800/52/r2/final), which requires TLS 1.3 support as of January 1, 2024.

There are two BoringCrypto entries [going through CMVP review](https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/modules-in-process-list) now:

|  |  |  |  |
| -- | -- | -- | -- |
| BoringCrypto | Google, LLC. | FIPS 140-3 | Review Pending (12/21/2023) |
| BoringCrypto | Google, LLC. | FIPS 140-3 | Coordination (12/14/2023) |

The "Coordination" one seems to be based on the `fips-20220613` tag.

As soon as the certificate is issued, I'll be updating all our docs and code to reference it (and reference FIPS 140-3 instead of 140-2).